contracts/Content/HasRoyalty.sol

```diff
@@ -56,7 +56,6 @@ abstract contract HasRoyalty is ContentSubsystemBase {
     /**
      * @dev Internal function to set the royalties for a token
      * @param _tokenId uint256 ID of the token to query
-     * If the token id does not have royalties, the contract royalties is returned.
```

Comment:
This should be "If the token id's royalty pair is not set (receiver = address(0)), then contract royalties is returned."

Reply:
Please add this back